### DISCUSSION TITLE
Removal of `show_sysload`

### DISCUSSION THREAD

**Discussion by UnrealKazu** [sabnzbd/sabnzbd#2222] [General]
I got a request for info regarding the removal of the `show_sysload` setting, as I've got it set to a non-default value (which in my case is `0`).

I've got it turned off because I'm not interested in seeing these kinds of system metrics _inside_ of SABnzbd. I've got other services that track these metrics for me and warn me if they are incorrect or have been high for too long. Because of this, I just turned it off altogether in SABnzbd as it was just a distraction for me.

It's a very minor feature tho, so if the setting were to be removed in favor of keeping the sysload indicator always on, that's fine by me as well.

**Comment by Safihre** [comment]
Thanks for letting us know. It could indeed stay as a switch instead of the multiple levels it has now.

**Comment by SoonGlob** [comment]
I got asked for the same feedback. I'm pretty much the same as OP. If i want to see this information I have better places to find it.

I prefer minimal UI, so I want to just disable this option.i'm maybe interested in RAM usage, but it doesn't actually matter as the system has more RAM than it ever tries to use.

**Reply by Safihre** [reply]
Thanks for letting us know!

**Comment by Legion-66** [comment]
The same as the other users, I am not interested in seeing performance information in the client. I can only think this has some overhead albeit slight compared to not displaying anything?

**Comment by JaySea77** [comment]
I received the message aswell. Same as the OP. I have tools to see the load, but I do not want that info in Sabnzbd.
